src/renderers/webvr/WebXRManager.js

```diff
@@ -97,6 +109,17 @@ function WebXRManager( renderer ) {
 
 	}
 
+	function onRequestFrameOfReference( value ) {
+
+		frameOfReference = value;
+
+		renderer.setFramebuffer( session.baseLayer.framebuffer );
```

Comment:
And here
`renderer.setFramebuffer( 'renderState' in session ? session.renderState.baseLayer.framebuffer : session.baseLayer.frameBuffer );`